cmd/plugin/purser_plugin.go

```diff
@@ -159,23 +129,54 @@ func createGroupNameFromLabel(input string) string {
 	return groupName
 }
 
-func init() {
-	kubeconfig := flag.String("kubeconfig", os.Getenv("KUBECTL_PLUGINS_GLOBAL_FLAG_KUBECONFIG"), os.Getenv("KUBECTL_PLUGINS_GLOBAL_FLAG_KUBECONFIG"))
-	flag.Parse()
-
-	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
-	if err != nil {
-		panic(err.Error())
+func computeStats(inputs []string) {
+	switch inputs[0] {
+	case Get:
+		getStats(inputs)
+	case Set:
+		setStats(inputs)
+	default:
+		printHelp()
 	}
+}
 
-	clientset, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		panic(err.Error())
+func getStats(inputs []string) {
+	if inputs[1] == "summary" {
+		plugin.GetClusterSummary()
+	} else if inputs[1] == "savings" {
+		plugin.GetSavings()
+	} else if inputs[1] == "user-costs" {
+		price := plugin.GetUserCosts()
+		fmt.Printf("cpu cost per CPU per hour:\t %f$\nmem cost per GB per hour:\t %f$\nstorage cost per GB per hour:\t %f$\n",
+			price.CPU,
+			price.Memory,
+			price.Storage)
+	} else {
+		printHelp()
 	}
-	plugin.ProvideClientSetInstance(clientset)
+}
 
-	// Crd client
-	crdclient = controller.GetAPIExtensionClient()
+func setStats(inputs []string) {
```

Comment:
I think we need to rename "setStats". Here we are saving user entered unit rates.